modules/sdk-coin-sui/src/lib/walrusStakingBuilder.ts

```diff
@@ -33,7 +33,7 @@ export class WalrusStakingBuilder extends TransactionBuilder<WalrusStakingProgra
     this._transaction = new StakingTransaction(_coinConfig);
 
     // TODO improve mainnet vs. testnet configuration
```

Comment:
was this TODO addressed via this PR , or are there further improvements to be made?

Reply:
I guess we can refactor this logic into the Sui utils in another pr

Reply:
please create a ticket and update the TODO comment to include that ticket so we dont forget to do this down the line